searchcore/src/vespa/searchcore/proton/matching/matching_stats.cpp

```diff
@@ -30,6 +30,7 @@ MatchingStats::MatchingStats()
       _doomOvertime(),
       _softDoomFactor(INITIAL_SOFT_DOOM_FACTOR),
       _queryCollateralTime(),
```

Comment:
Add 'TODO Vespa 8' where appropriate, either here or somewhere to make this line fail compilation when the metric is removed.